fenix/app/src/main/java/org/mozilla/fenix/messaging/MessageNotificationWorker.kt

```diff
@@ -223,7 +223,10 @@ class NotificationClickedReceiverActivity : Activity() {
             runBlockingIncrement { nimbusMessagingController.onMessageClicked(message.metadata) }
 
             // Create the intent.
-            val intent = nimbusMessagingController.getIntentForMessageAction(message.action)
+            val intent = Intent(
+                Intent.ACTION_VIEW,
+                nimbusMessagingController.processMessageActionToUri(message),
+            )
```

Comment:
Nit: can we have a `getIntentForMessage(message)` which does all this for you?

Reply:
Sure. I did remove that as a func but have just updated to call processMessageActionToUri